queue_services/entity-emailer/src/entity_emailer/email_templates/NR-PAID.html

```diff
@@ -37,7 +37,7 @@
             PLEASE DO NOT REPLY TO THIS EMAIL. It was sent from an unmonitored email address and the Corporate Registry is unable to respond to any replies.
 
             [[whitespace-16px.html]]
-            [[nr-footer.html]]            
+            [[nr-footer-html.html]]
```

Comment:
This is a different filename from the other templates above. Typo?

Reply:
This one is still using HTML format template because it has attachment on it. So, have to use different footer.